build.sbt

```diff
@@ -0,0 +1,196 @@
+// Configurable settings
+val scalaJSVersion =
+  settingKey[String]("Version of Scala.js for which to build to CLI")
+val scalaJSScalaVersions =
+  settingKey[Seq[String]]("All the minor versions of Scala for which to build the CLI")
+
+// Derived settings
```

Comment:
Maybe "computed" settings? Derived actually means something in sbt land :-/

Reply:
Good point. I'll change it.